src/config.js

```diff
@@ -1619,9 +1619,9 @@ export const CONFIG = {
       platform: 'windows10-64',
       benchmarks: DEFAULT_SUITES,
     },
-    win10Laptops: {
-      label: 'Windows 10 64bit (laptops)',
-      platform: 'windows10-64-ux',
+    win10ref2017: {
```

Comment:
Can we redirect users from the old URL?

Reply:
You can use this:
https://reacttraining.com/react-router/web/api/Redirect